OutOfSchool/OutOfSchool.RazorTemplatesData/Views/Emails/ApplicationStatusUpdateEmail/ApplicationStatusUpdateEmail.cshtml

```diff
@@ -0,0 +1,57 @@
+﻿@using OutOfSchool.RazorTemplatesData.Models.Emails
+@using OutOfSchool.RazorTemplatesData.Models.Shared
+@using OutOfSchool.Services.Enums
+@model ApplicationStatusViewModel
+
+@{
+    ViewData["EmailTitle"] = "";
+    var ending = "";
+
+    if (Model.ChildGender == Gender.Male) 
+    {
+        ending = "ий";
+    }
+    else if (Model.ChildGender == Gender.Female)
+    {
+        ending = "а";
+    }
+    else if (Model.ChildGender == null)
+    {
+        ending = "ий/a";
+    }
+}
+
+<main class="container">
+    <div class="content">
+        @if (Model.ApplicationStatus == ApplicationStatus.Approved)
+        {
+            <h1 class="title">Зараховано!</h1>
```

Comment:
(Please teach me ;) ) What is our current policy about localizations? Because here everything is hardcoded, though I see some localized resources in files below.

Reply:
@VadymLevkovskyi, I asked @DmyMi if I should implement localizations of email texts. Dmytro offered to make localizations in a separate PR, because we need to do localizations not only in this email, but also in other emails.

Reply:
Thanks for the info. Ok, let's have it introduced in next PR